bin/ycsb.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/sh
```

Comment:
I love bash, but when it comes to shell writing, I go old school.  POSIX sh seems to be the choice with most delivered software that I see.  Perhaps it's time for a change because as you said, bash is very prevalent.  I can change it.